jcli/src/jcli_app/utils/debug_flag.rs

```diff
@@ -9,10 +11,24 @@ pub struct DebugFlag {
 }
 
 impl DebugFlag {
```

Comment:
The type name does not match the purpose, rename to `DebugWriter` or something like that?

Reply:
It's still primarily a flag passed from CLI, which happens to encapsulate the logic of what this flag is supposed to do. If this argument doesn't convince you, I'll rename it.

Reply:
Ah, I didn't see how it was the option value.